packages/micropip/src/micropip/_micropip.py

```diff
@@ -280,8 +280,8 @@ async def add_requirement(
                     f"but {req.name}=={ver} is already installed"
                 )
         metadata = await _get_pypi_json(req.name)
-        wheel, ver = self.find_wheel(metadata, req)
-        if wheel is None and ver is None:
+        maybe_wheel, maybe_ver = self.find_wheel(metadata, req)
+        if maybe_wheel is None or maybe_ver is None:
```

Comment:
This is a minor change (and to or) to make typing happy. I think they were always in sync.